frontend/packages/metal3-plugin/src/selectors/baremetal-hosts.ts

```diff
@@ -24,6 +24,8 @@ export const getHostMachineName = (host: BareMetalHostKind): string =>
   _.get(host, 'spec.consumerRef.name');
 export const getHostBMCAddress = (host: BareMetalHostKind): string =>
   _.get(host, 'spec.bmc.address');
+export const getHostBootMACAddress = (host: BareMetalHostKind) =>
+  _.get(host, 'spec') && host.spec.bootMACAddress;
```

Comment:
Why not this? `_.get` returns `undefined` if one of the items in the path does not exist.
```suggestion
  _.get(host, 'spec.bootMACAddress');
```

Reply:
because the type remains this way

Reply:
Fair, it's IMHO quite ugly workaround though. We could start using typescript's optional chaining for these cases. (Requires prettier update)

Reply:
relevant open PR: https://github.com/openshift/console/pull/3747